rero_ils/modules/selfcheck/api.py

```diff
@@ -350,65 +355,68 @@ def selfcheck_checkout(user_pid, institution_id, patron_barcode,
                                 .append(checkout_note)
                         # TODO: When is possible, try to return fields:
                         #       magnetic_media, desensitize
-                except ItemNotAvailableError:
-                    checkout.get('screen_messages').append(
-                        _('Checkout impossible: the item is requested by '
-                          'another patron'))
-                except Exception:
-                    checkout.get('screen_messages').append(
-                        _('Error encountered: please contact a librarian'))
+        except ItemNotAvailableError:
+            checkout.get('screen_messages').append(
```

Comment:
`checkout.get('screen_messages', []).append(` to be sure ? maybe not necessary.

Reply:
ok, done.